mixins/localize-mixin.md

```diff
@@ -121,6 +121,65 @@ If your localized string contains arguments, pass them as a key-value object as
 
 ```javascript
 render() {
-  return html`<p>${this.localize('hello', {firstName: 'Mary'})}</p>`;
+  return html`<p>${this.localize('hello', { firstName: 'Mary' })}</p>`;
 }
 ```
+
+## `localizeHTML()`
+
+You can include some basic markup in your localization resources and safely convert them to HTML with the `localizeHTML()` method.
+
+### Simple Markup
+
+#### Bold
+
+The `[b]bold[/b]` markup converts to the `<strong>` tag.
+
+#### Italic
+
+The `[i]italic[/i]` markup converts to the `<em>` tag.
+
+#### Line Breaks
+
+The `[br]` markup converts to the `<br>` tag.
```

Comment:
Have paragraphs `<p>` been considered? Paragraph elements support styling in ways that `<br>` does not - spacing is the one that generally comes to mind. I think screen readers can also have short-cuts for moving the virtual cursor between paragraphs. tinyMCE switched to paragraphs 10+ years ago so all of our user-authored HTML generally consists of them.